DESCRIPTION

```diff
@@ -22,7 +22,8 @@ Imports:
     deSolve,
     stats,
     poweRlaw,
-    methods
+    methods,
+    usethis
```

Comment:
I think this not need. See blow.

Otherwise it should be in Suggests and not the imports. My assuimption would be that you don't use anywhere else.